### ISSUE TITLE
OEM locking option set to "off" and greyed out, but is_oem_unlocked set to true

### ISSUE THREAD

**Issue by shamen123** [GrapheneOS/os_issue_tracker#61]
As noted here, I have had an issue with the OEM unlocking switch since flashing Graphene OS
https://www.reddit.com/r/GrapheneOS/comments/bo1fhf/pixel2_oem_unlock

Version: PQ3A.190505.001.2019.05.08.15
Build type: Official stable
Device: Pixel2 XL

Summary: The Developer option OEM unlocking is set to "off" and greyed out, however fastboot var is_oem_unlocked is set to true and device does respond to fastboot flashing unlock/lock commands

Steps to reproduce: Using Pixel2 XL, enable developer and enable OEM unlock. Enter bootloader, fastboot flashing unlock, extract graphene OS stable PQ3A.190505.001.2019.05.08.15, flash-all.sh, reboot to bootloader. Perform fastboot flashing lock and boot. Enter developer tools, turn OEM flashing to off. reboot. OEM flashing is both set to off and also greyed out in developer tools, but is_oem_unlocked is set true. Attempt 'fastboot flashing unlock' and see bootloader unlocks. 

Will add a video shortly

**Comment by shamen123** [comment]
https://www.youtube.com/watch?v=K0vC-wT0rmE

**Comment by shamen123** [comment]
I tried unlocking and re-flashing Graphene but got the same result. This has been resolved by flashing to latest stock firmware, then re-flashing back to grapheneOS. I can only assume that the original bootloader that was on the device when supplied to me was too old (eg, required critical unlock). Though its weird I got no errors when doing the first flash. Either way, flashing to latest android first has resolved it.
